controller/controller.js

```diff
@@ -68,3 +70,12 @@ exports.getUsers = (req, res, next) => {
     res.status(200).send(users);
   });
 };
+
+
+exports.deleteComment = (req,res,next) => {
+  removeComment(req.params.comment_id).then((deletedComment)=> {
+    res.status(204).send(deletedComment)
```

Comment:
Just respond with the 204 - and don't send anything back at all. Change to .send().